docs/source/notebooks/WCS_example.ipynb

```diff
@@ -31,22 +31,22 @@
      "name": "stdout",
```

Comment:
<p>The colors of the new graph is fairly different !  Is this normal?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/pavics-sdi/pull/324/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>It could be that the data no longer has a <code>NaN</code> value. It's probably fine. </p>